lib/ui/painting/image_decoding.cc

```diff
@@ -34,24 +34,13 @@ sk_sp<SkImage> DecodeImage(sk_sp<SkData> buffer) {
     return nullptr;
   }
 
-  auto raster_image = SkImage::MakeFromEncoded(std::move(buffer));
-
-  if (raster_image == nullptr) {
-    return nullptr;
-  }
-
-  if (auto context = ResourceContext::Get()) {
-    // TODO: Supply actual destination color space once available
-    if (auto texture_image =
-            raster_image->makeTextureImage(context, nullptr)) {
-#ifdef OS_ANDROID
```

Comment:
There is an import for `GLES2/gl2.h` on top that you should probably get rid of too.